lib/src/log.rs

```diff
@@ -129,22 +130,84 @@ pub fn reset_task_log_directory(path: &Path) -> Result<(), Error> {
     Ok(())
 }
 
+pub fn read_last_lines_as_byte_deque(file: &mut File, lines: usize) -> Result<VecDeque<u8>, Error> {
+    // The integer types were chosen to avoid overflows when converting between signed and unsigned
+    // integers. u16 is used here so that we can support 16bit platforms with simple integer casts
+    // (FIXME Not sure if pueue even is supported on any 16bit platforms)
+    let chunk_size = 4096u16;
```

Comment:
Pueue isn't used on 16bit platforms. At least I'm not aware of it.
Anyhow, u64 should still be supported u16 systems anyway. There are also u128 on 64-bit systems. The number will then just occupy four memory addresses instead of a single one.

I think it's best to switch this to `u64`. This will remove a lot of number casting and error handling.

Reply:
If I used `u64` though, I believe I would need to do `try_into` or something similar on 32-bit platforms to be safe for the cast to `usize` (obviously the chunk size won't be that big). Thats what I was trying to avoid. Though its possible all my casting is less readable than using `try_into` . So even if we drop 16-bit, to keep my code as is, it could only be `u32`.

Reply:
Ah I see. You're right about that. Casting to usize wouldn't be safe in that case.